### ISSUE TITLE
Prepare plugins again for Matomo 4 release

### ISSUE THREAD

**Issue by tsteur** [matomo-org/matomo#16623]
* [x] For each public plugin in matomo and innocraft namespace make sure 3.x was merged into 4.x
* [x] Check there is a release for each plugin. We had already released an update for each plugin but some had more changes and we need to do one more release
* [x] Update the link in the readme to travis as repository was likely moved to travis com
* [x] Check all submodules etc are up to date in 4.x-dev branch
* [x] Ideally we also check once more the tests are green for our plugins.
* [x] We also need to do all above for the premium features. Once all is done I can release the new versions on the marketplace as some had changes

**Comment by sgiehl** [comment]
I've looked through all plugins in matomo and innocraft space and merged `3.x-dev` to `4.x-dev` where `3.x-dev` was ahead of `4.x-dev` (except of Cloud and Account2)

**Comment by tsteur** [comment]
Thanks @sgiehl . I'll try to go over all the plugins and release new versions were needed
